buildSrc/src/main/kotlin/mockito.publication-conventions.gradle.kts

```diff
@@ -0,0 +1,164 @@
+import gradle.kotlin.dsl.accessors._2a0859b2b1624a8c1f98b2dbe4a3b98d.javadoc
+import org.gradle.api.plugins.JavaPlugin
+import org.gradle.kotlin.dsl.`maven-publish`
+import org.gradle.kotlin.dsl.withType
+
+plugins {
+    `maven-publish`
+    signing
+}
+
+// enforce the same group for all published artifacts
+group = "org.mockito"
+
+// generic publication conventions
+publishing {
+    publications {
+        withType(MavenPublication::class).configureEach {
+            pom {
+                artifactId = project.base.archivesName.get()
+                name = artifactId
+                afterEvaluate {
```

Comment:
Same concern with `afterEvaluate` here. Can we instead define a `mockitoPublicationExtension` to configure the description?

Reply:
This was in the original script, let's see if it's necessary now that it is in a convention plugin.

Reply:
Done